core/shared/src/main/scala/plotly/layout/Legend.scala

```diff
@@ -15,7 +15,8 @@ import dataclass.data
   bordercolor: Option[Color],
       bgcolor: Option[Color],
       xanchor: Option[Anchor],
-      yanchor: Option[Anchor]
+      yanchor: Option[Anchor],
+  orientation: Option[String]
```

Comment:
[`Orientation`](https://github.com/alexarchambault/plotly-scala/blob/89a2c04407972e234fd8c131e116b737ac8b88a3/core/shared/src/main/scala/plotly/element/Orientation.scala) could be used here instead of `String`.